app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -61,7 +61,8 @@ miqHttpInject(angular.module('containerDashboard', ['ui.bootstrap', 'patternfly'
           id = '';
         }
         else {
-          id = '/'+ (/\/(\d+)/.exec($location.absUrl())[1]);
+          // search for http://<url>/<controller>/<number>...
+          id = '/' + (/\/.+\/.+\/(\d+)/.exec($location.absUrl())[1]);
```

Comment:
@himdel Thanks, fixed to anchor and pathname like you suggested  :cat2: 
`/^\/[^\/]+\/(\d+)$/.exec(location.pathname)`